utils/node_local_test.py

```diff
@@ -414,6 +414,7 @@ def get_base_env(clean=False):
     env['D_LOG_MASK'] = 'DEBUG'
     env['D_LOG_SIZE'] = '5g'
     env['FI_UNIVERSE_SIZE'] = '128'
+    env['DAOS_STRICT_SHUTDOWN'] = '1'
```

Comment:
If this is for the server only then https://github.com/daos-stack/daos/blob/master/utils/nlt_server.yaml would probably be a better place to add it.

Reply:
Sure, will move it there.